src/lib/components/Banner.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+	export let message = 'Welcome to our website!'
+	export let bgColor = '#4CAF50' // Default background color
+
+	let visible = true // Initially, the banner is visible
+</script>
+
+{#if visible}
+	<div class="banner" style="--bg-color: {bgColor};">
+		<span>{@html message}</span>
+		<button class="close-btn" on:click={() => (visible = false)}>×</button>
+	</div>
+{/if}
+
+<style>
+	.banner {
+		padding: 16px;
+		background-color: var(--bg-color);
+		color: var(--black);
+		font-family: var(--font-heading);
+		font-weight: bold;
+		text-align: center;
+		font-size: 1.25em;
+		position: relative;
+	}
+
+	:global(.banner a) {
+		font-family: var(--font-heading);
+		/* font-weight: 700; */
+		color: var(--white);
```

Comment:
White on orange is not enough contrast. You can check this through the dev tools, [for example](https://firefox-source-docs.mozilla.org/devtools-user/accessibility_inspector/#color-contrast). Probably, we need some other way of making the link obvious. Maybe a simple underline is enough, or we could remove bold on the rest of the text and keep it only for links, or add italics for the link. Do some tests to see what looks best.

Reply:
done